app/views/categories/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= stylesheet_link_tag 'categories', media: 'all', 'data-turbolinks-track': 'reload' %>
+
+<div class ="categories">
+  <%= link_to "New category", new_category_path, class: "link_to newcat" %>
```

Comment:
- [ ] As requested, please move this button to the bottom of the page to match the design.
![image](https://github.com/tjay1760/Budget-App/assets/91486791/164ff739-fb9d-4c7c-8f55-a555fec4fcc1)